hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -4,22 +4,20 @@
 {
   name: "chip"
 
-  // Replicate CSR and mem tests over TL and JTAG interfaces.
-  intf: ["_tl", "_jtag"]
-
   // TODO: remove the common testplans if not applicable
   import_testplans: ["hw/dv/tools/dvsim/testplans/csr_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/enable_reg_testplan.hjson",
-                     "hw/dv/tools/dvsim/testplans/mem_testplan.hjson",
+                     // TODO #5484, comment these 2 lines out because spi host memory is dummy
```

Comment:
Should the priority of issue #5484 be raised?

Reply:
I have contacted Martin to updated the bug with status offline.